common/src/main/java/com/amazonaws/services/schemaregistry/common/AWSSchemaRegistryClient.java

```diff
@@ -90,11 +91,26 @@ public AWSSchemaRegistryClient(@NonNull AwsCredentialsProvider credentialsProvid
                 .addExecutionInterceptor(new UserAgentRequestInterceptor())
                 .build();
 
+        UrlConnectionHttpClient.Builder urlConnectionHttpClientBuilder = UrlConnectionHttpClient.builder();
+
+        if (glueSchemaRegistryConfiguration.getProxyUrl() != null) {
+        	log.debug("Creating http client using proxy {}", glueSchemaRegistryConfiguration.getProxyUrl());
+        	try {
+    	    	URI uri = new URI(glueSchemaRegistryConfiguration.getProxyUrl());
+    	        ProxyConfiguration proxy = ProxyConfiguration.builder().endpoint(uri).build();
+    	        urlConnectionHttpClientBuilder.proxyConfiguration(proxy);
+        	} catch (URISyntaxException e) {
```

Comment:
You might want to consider moving this to validation to `GlueSchemaRegistryConfiguration.java` similar to how it is being done for the configuration parameters.

Reply:
@blacktooth  moved the validation to GlueSchemaRegistryConfiguration.java